pinot-plugins/pinot-input-format/pinot-csv/src/test/java/org/apache/pinot/plugin/inputformat/csv/CSVRecordReaderTest.java

```diff
@@ -103,4 +104,17 @@ protected void checkValue(RecordReader recordReader, List<Map<String, Object>> e
     }
     Assert.assertFalse(recordReader.hasNext());
   }
+
+  @Test
+  public void whenConfiguredHeaderDoesNotContainConfiguredDelimiterThenExceptionShouldBeThrown() {
```

Comment:
please use a short test name (testIncorrectHeaderDelimiter ?) and add the javadoc to this method if you want to have this description somewhere

Reply:
Yes, actually I tend to name UTs in givenWhenThen format. This also ensures people are not asserting on many different conditions. Will change to keep consistent with Pinot's format.